chat.sympl

```diff
@@ -525,50 +525,27 @@ def _get_messages(room_channel: ChannelName) -> List[Message]:
     if isinstance(room, Room):
         if room.is_deleted:
             cvm.error(f"Room {room_channel} has been deleted. Cannot get messages.")
-        return cvm.storage.query_history(MessageStatic).in_channel(room_channel).order_by('timestamp', desc=True).values()
+        return cvm.storage.query_history(MessageStatic).in_channel(room_channel).order_by('timestamp', False).values()
     else:
         cvm.error(f"Room for channel {room_channel} not found.")
 
 @clientside_helper
 def _get_rooms() -> List[Room]:
     ##this function gets the most recent readable version of the room by the caller (i.e. this gets the current version
     # if you are still in the room, and if you were kicked out, the version immediately after you were kicked out)
+    results = cvm.storage.query_history(RoomStatic).where('is_deleted', False).order_by('channel', False).order_by('name', False).values()
```

Comment:
the original code had an explicit type cast for the variable `results`. Why did we remove it?